browser/ai_chat/upload_file_helper.cc

```diff
@@ -136,27 +168,26 @@ void UploadFileHelper::FileSelected(const ui::SelectedFileInfo& file,
     observer.OnFilesSelected();
   }
   profile_->set_last_selected_directory(file.path().DirName());
-  auto read_image = base::BindOnce(
+  auto read_file = base::BindOnce(
       [](const ui::SelectedFileInfo& info)
-          -> std::tuple<std::optional<std::vector<uint8_t>>, std::string> {
-        return std::make_tuple(
-            ai_chat::ReadFileToBytes(info.path()),
-            base::FilePath(info.display_name).AsUTF8Unsafe());
+          -> std::tuple<std::optional<std::vector<uint8_t>>, base::FilePath> {
+        return std::make_tuple(ai_chat::ReadFileToBytes(info.path()),
```

Comment:
nit: You could deduplicate this lambda in a static function

Reply:
fixed in db052348920037997bd564c04bf5d4d6505199bf